ncutixavier/my-brand
# #A9AM8NNu Implementing landing page

#### What does this PR do?
This PR implements a landing page

#### Description of Task to be completed?
- Add homepage
- Update about page

#### How should this be manually tested?
- Clone the [repo](https://github.com/ncutixavier/my-brand)
- Checkout branch `feat-implement-landing-page-A9AM8NNu`
- Open the index.html in your browser to check the new homepage

#### What are the relevant pivotal tracker stories?
[#A9AM8NNu](https://trello.com/c/A9AM8NNu/53-implement-landing-page)

#### Screenshots
N/A
